apps/nextjs-website/src/components/VideoImage.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+import { VideoImage as VideoImageRC } from '@react-components/components';
+import { VideoImageProps } from '@react-components/types';
+import { VideoImageSection } from '@/lib/fetch/types/PageSection';
+
+const makeVideoImageProps = ({
+  src,
+  alt,
+  ...rest
+}: VideoImageSection): VideoImageProps => ({
+  ...rest,
+  ...(src.data && {
+    src: {
+      url: src.data.attributes.url,
+      mime: src.data.attributes.mime,
+    },
+  }),
+  alt,
+});
```

Comment:
```suggestion
const makeVideoImageProps = ({
  src,
  ...rest
}: VideoImageSection): VideoImageProps => ({
  ...rest,
  ...(src.data && {
    src: {
      url: src.data.attributes.url,
      mime: src.data.attributes.mime,
      ...(src.data.attributes.alternativeText && { alt: src.data.attributes.alternativeText }),
    },
  }),
});

```

Reflect suggested change in type, moving alt inside src